src/NServiceBus.RavenDB.Tests/API/APIApprovals.cs

```diff
@@ -1,27 +1,18 @@
 ﻿namespace NServiceBus.RavenDB.Tests.API
 {
-    using System.IO;
     using System.Runtime.CompilerServices;
-    using ApiApprover;
-    using ApprovalTests;
-    using ApprovalTests.Reporters;
-    using Mono.Cecil;
     using NUnit.Framework;
+    using PublicApiGenerator;
 
     [TestFixture]
     class APIApprovals
     {
         [Test]
         [MethodImpl(MethodImplOptions.NoInlining)]
-        [UseReporter(typeof(DiffReporter))]
         public void ApproveRavenDbPersistence()
         {
-            Directory.SetCurrentDirectory(TestContext.CurrentContext.TestDirectory);
-            var assemblyPath = Path.GetFullPath(typeof(RavenDBPersistence).Assembly.Location);
-            var asm = AssemblyDefinition.ReadAssembly(assemblyPath);
-            var publicApi = PublicApiGenerator.CreatePublicApiForAssembly(asm, definition => true, false);
-
-            Approvals.Verify(publicApi);
+            var publicApi = ApiGenerator.GeneratePublicApi(typeof(RavenDBPersistence).Assembly, excludeAttributes: new[] { "System.Runtime.Versioning.TargetFrameworkAttribute" });
```

Comment:
You could leave off the `excludeAttributes: new[] { "System.Runtime.Versioning.TargetFrameworkAttribute" }` since it would be okay to have that in the approval file for this branch.